components/formik-map.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+import { Map, CircleMarker, TileLayer } from 'react-leaflet'
+import { reverse } from 'ramda'
+
+export default class FormikMap extends Component {
```

Comment:
This should probably just be `Map` as formik isn't used. Or maybe `InputMap` of `FormMap` or something like that that specifies it's meant for user input.